### PR TITLE
Send post Markdown from SE API to MS

### PR DESCRIPTION
**Pull request by iBug**
Context: Charcoal-SE/metasmoke#734

This PR implements the Smokey side of stuff. Apparently depends on the deployment of the target MS PR.

---

Changes to filters: Added `body_markdown` for questions and answers where `body` is already selected.

### CODE REVIEW

**Comment by iBug** [comment]
cc @ArtOfCode- Ready for merge after MS deploy

File: chatcommands.py [hunk 1]
```diff
@@ -1731,7 +1731,7 @@ def allspam(msg, url):
         # Fetch posts
         request_url = "https://api.stackexchange.com/2.2/users/{}/posts".format(u_id)
         params = {
-            'filter': '!)Q4RrMH0DC96Y4g9yVzuwUrW',
+            'filter': '!*0S0--txpIwJHBnEqJR9px_-b(4TI_dAeSMiPUcCA',
```
Comment (makyen) [inline chatcommands.py:1734]: This filter does not appear to contain `post/body_markdown`, which is what you are trying to get here, at least as far as I understand.

Comment (iBug) [reply chatcommands.py:1734]: Oops, I only looked into `answer` and `question` and even wondered for a while why it included neither `answer/body` nor `question/body`.

Comment (iBug) [reply chatcommands.py:1734]: Fixed in abd4b5011ee82b206586411b11b616ca7fdb832f

**Pull request merged**